launch/hero_bringup.launch

```diff
@@ -21,6 +21,7 @@
     <arg name="robot_name" default="hsrb" />
     <arg name="personal_name" default="hero" />
     <arg name="enable_emergency_stop" default="false" />
+    <arg name="gui" default="false" />  <!-- Gazebo GUI -->
```

Comment:
Call it `gazebo_gui` as just `gui` is not clear

Reply:
done